src/periphery/CoolerV2Migrator.sol

```diff
@@ -90,6 +89,9 @@ contract CoolerV2Migrator is
     /// @notice The Cooler V2 contract
     IMonoCooler public immutable COOLERV2;
 
+    /// @notice The CoolerFactory contract
+    ICoolerFactory public immutable COOLER_FACTORY;
```

Comment:
Isn't this a different factory contract for v1.1 vs v1.2/v1.3 ?

Reply:
You're right. Good find. I'll add support for both.